src/components/navbar/Navbar.js

```diff
@@ -2,8 +2,10 @@ import React from "react";
 import "./../../App.css";
 import { Link } from "react-router-dom";
 import storeLogo from "../../assets/location.png";
-
+import { useCart } from "../../context/cart-context";
 export default function Navbar() {
+  const { stateCart } = useCart();
+  const { cart } = stateCart;
```

Comment:
you can do the same in single line as `const { stateCart : { cart } } = useCart();` in future

Reply:
ohh wow ! Thanks for suggestion